androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/CloseButton.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+
+@Composable
+fun CloseButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Image(
+        painter = painterResource(id = R.drawable.ic_close),
+        contentDescription = null,
+        modifier = modifier
+            .size(28.dp)
```

Comment:
I think we can make a constant for this `28.dp` for both close and back button. Also are you planning to reuse these close and back button for other screens?

Reply:
Removed the file and moved the `Image` to the caller. https://github.com/nimblehq/avishek-phong-kmm-ic/pull/120/commits/f686f7addab4416fb526524b008aed5a74ee2412